integration_tests/android_support_test/src/test/java/org/robolectric/integration_tests/atsl/robolectric.properties

```diff
@@ -0,0 +1,2 @@
+sdk=ALL_SDKS
+packageName=org.robolectric.integration_tests.atsl
```

Comment:
Not sure you need to specify the package name do you? I didn't need to for ctesque